src/main/java/com/openshift/jenkins/plugins/util/ClientCommandBuilder.java

```diff
@@ -111,6 +106,14 @@ private boolean hasArg(List<String> args, String... argsToFind) {
                 cmd.add("--token=" + token);
             }
         }
+        
+        cmd.add(verb);
+
+        cmd.addAll(toStringArray(verbArgs));
+
+        cmd.addAll(toStringArray(userArgs));
+
+        cmd.addAll(toStringArray(options));
```

Comment:
this is just ensuring that the "--" arg does not come before the "--loglevel" or other args being added?

Reply:
ah ... i should update the comment, forgot that the literal `--` sequence could be a valid arg

what I was trying to convey was that things like --server, --namespace, --token, --loglevel in the wrong position would confuse `oc rsh` ... that is what was causing the problem .... moving those args prior to the verb works generically, and protects `oc rsh`

Reply:
got it, yeah a clarifying comment would help, otherwise lgtm.

Reply:
comment clarified